beacon_node/eth2-libp2p/src/rpc/methods.rs

```diff
@@ -205,6 +211,7 @@ impl<T: EthSpec> RPCErrorResponse<T> {
                 RPCResponse::Status(_) => false,
                 RPCResponse::BlocksByRange(_) => true,
                 RPCResponse::BlocksByRoot(_) => true,
+                RPCResponse::Testing(_) => true,
```

Comment:
Probably remove this an use a full block for these kinds of tests, before merging